pkg/mq/mqimpl/rocksmq/client/client_impl.go

```diff
@@ -191,6 +198,17 @@ func (c *client) tryToConsume(consumer *consumer) []*RmqMessage {
 	}
 	rmqMsgs := make([]*RmqMessage, 0, len(msgs))
 	for _, msg := range msgs {
+		rmqMsg, err := unmarshalStreamingMessage(consumer.topic, msg)
+		if err == nil {
```

Comment:
The error code makes it difficult to determine whether the message is from the streaming node; checking the message source directly is clearer.